index.js

```diff
@@ -120,9 +137,25 @@ async function print(argv) {
     await page.goto(url, buildNavigationOptions(argv));
 
     console.error(`Writing ${argv.output || 'STDOUT'}`);
+
+    let height, width;
+    if (argv.magicformat) {
+        height = (await page.evaluate(
+            'Math.max(document.body.scrollHeight, document.body.offsetHeight, document.documentElement.clientHeight, document.documentElement.scrollHeight, document.documentElement.offsetHeight)'));
+        width = '1366';
+        argv.format = undefined;
```

Comment:
let's treat `argv` as immutable, either vary what gets passed to `page.pdf(...)` with ternary operators or pull out a mutable variable to be built up procedurally like you did with `width` and `height`

Reply:
done